portal/patient_self_checkin/index.php

```diff
@@ -0,0 +1,120 @@
+<?php
+/**
+ * Patient Self Check in
+ *
+ * This program allows patients to check themselves in using a touchscreen or similar device in the doctors' practice.
+ * Doing so, they are marked as arrived (code @) on the calendar module of the clinician's screen and are also marked
+ * as present in the Patient Flow Board.
+ * The purpose of this program is to free up time for front desk staff.
+ *
+ * @category  Portal
+ * @package   OpenEMR
+ * @author    Alfie Carlisle <asc@carlisles.co>
+ * @copyright Copyright (c) 2017 Alfie Carlisle <asc@carlisles.co>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ * @link      http://www.open-emr.org
+ */
+
+$ignoreAuth = true;
+require_once "../../interface/globals.php";
+require_once $GLOBALS['srcdir'].'/sql.inc';
+if ($GLOBALS['self_checkin_enable']==1) {
+?>
+
+<!-- Latest compiled and minified CSS -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
```

Comment:
authentication is getting bypassed for now via the `$ignoreAuth = true`
(still thinking about the best way to deal with authentication since would be ideal to sandbox this like the portals do)